iceoryx_hoofs/test/moduletests/test_cxx_functional_interface_types.cpp

```diff
@@ -97,4 +97,66 @@ OptionalFactory::Type OptionalFactory::createInvalidObject() noexcept
 ///////////////////////
 /// END OptionalFactory
 ///////////////////////
+
+//////////////////////////////////
+/// BEGIN ExpectedValueErrorFactory
+//////////////////////////////////
+ExpectedValueErrorFactory::value_t ExpectedValueErrorFactory::usedTestValue = 11;
+ExpectedValueErrorFactory::value_t ExpectedValueErrorFactory::anotherTestValue = 12;
+ExpectedValueErrorFactory::error_t ExpectedValueErrorFactory::usedErrorValue = 13;
+ExpectedValueErrorFactory::error_t ExpectedValueErrorFactory::anotherErrorValue = 14;
+
+void ExpectedValueErrorFactory::configureNextTestCase() noexcept
+{
+    // we increment all the values with an arbitrary value in every test case
```

Comment:
The values are not arbitrary. Did you initially meant to use random values here?

Reply:
What about `s/arbitrary value/static arbitrary value` then? This would make it clearer for me.